### PR TITLE
Create CI/CD Pipeline for VS Code Extension Project

### PR DESCRIPTION
**Pull request by nagarjunsanji**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+## Prerequisites for merge, code author responsibilities
+* [ ] I as the code author have ensured that acceptance criteria is met
+* [ ] I as the code author have self-reviewed code and functionality
+* [ ] I have added new tests/extended existing ones
+* [ ] If this contains frontend changes, a design review has been requested and completed
+* [ ] A security review is initialized according to [Security Chapter Policy](https://www.notion.so/Secure-code-review-75585ae5c1ea43dab08f3a390714afb5)
```
Comment (emilwareus) [inline .github/pull_request_template.md:6]: We can skip this one as it is an internal document

File: README.md [hunk 2]
```diff
@@ -1,71 +1,114 @@
...
+- Inspired by the Visual Studio Code Extension Guide.
 
-* [Visual Studio Code's Markdown Support](http://code.visualstudio.com/docs/languages/markdown)
-* [Markdown Syntax Reference](https://help.github.com/articles/markdown-basics/)
+## Contributors
+We appreciate all the Contributors who have helped make this project better.
 
-**Enjoy!**
+## Milestones
+Check out our Milestones to see our project roadmap and upcoming features.
```
Comment (emilwareus) [inline README.md:114]: We don't have any open milestones etc here yet. Maybe skip this part?

**Review by emilwareus — CHANGES_REQUESTED** [review]
Well done, just fix some small changes and I'll approve it

File: README.md (hunk 2 above)
Comment (nagarjunsanji) [reply README.md:114]: removed Milestones as part of Update : boilerplate VS Code Extension #2 PR

File: .github/pull_request_template.md (hunk 1 above)
Comment (nagarjunsanji) [reply .github/pull_request_template.md]: Fixed

**Pull request merged**